language/de/ucp.php

```diff
@@ -607,7 +607,7 @@
 	'UNWATCH_MARKED'				=> 'Ausgewählte Abonnements beenden',
 	'UPLOAD_AVATAR_FILE'			=> 'Von deinem Rechner hochladen',
 	'UPLOAD_AVATAR_URL'				=> 'Von URL hochladen',
-	'UPLOAD_AVATAR_URL_EXPLAIN'		=> 'Gib die URL an, an der sich das Avatar-Bild befindet. Es wird dann auf diese Site hochgeladen.',
+	'UPLOAD_AVATAR_URL_EXPLAIN'		=> 'Gib die URL an, an der sich das Avatar-Bild befindet. Es wird dann auf diese Site übertragen.',
```

Comment:
Das war meiner Ansicht nach generell falsch ("Enter the URL of the location containing the image. The image will be copied to this site.")